packages/components/src/palette-edit/index.js

```diff
@@ -197,6 +197,9 @@ function PaletteEditListView( {
 			}
 		};
 	}, [] );
+
+	const debounceOnChange = useCallback( debounce( onChange, 200 ), [] );
```

Comment:
Would it be worth using the [useDebounce](https://github.com/wordpress/gutenberg/blob/bdb92729488eb33aed0b2ed949e9236153911f06/packages/compose/src/hooks/use-debounce/index.js#L28) hook from the `@wordpress/compose` package? (It's effectively pretty much the same, but a small abstraction)

Reply:
Nice, didn't know about useDebounce - have switched to that is already memoed so no need for useCallback then